translations/handleDesktopTranslations.sh

```diff
@@ -12,16 +12,58 @@ gpg --list-keys
 git clone git@github.com:nextcloud/desktop.git
 cd desktop
 
-lupdate-qt5 src/gui/ src/cmd/ src/common/ src/crashreporter/ src/csync/ src/libsync/ -ts translations/client_en.ts
+# Generate source translation files for master and stable branches
+mkdir /branches
+
+versions="stable-2.6 master"
+if [[ -f '.tx/backport' ]]; then
+  versions="$(cat .tx/backport) master"
+fi
+
+for version in $versions
+do
+  # skip if the branch doesn't exist
```

Comment:
If you get the branches dynamically via git, you can skip this

Reply:
Not if we specify them in .tx/backport and someone makes a mistake ;p